projects/vdk-plugins/VEP-Universal Database Enhancement Proposal/Universal_Database_Enhancement_Proposal_Draft.txt

```diff
@@ -0,0 +1,109 @@
+q# VEP-NNNN: Universal Database Enhancement Proposal
+
+* **Author(s):** Maximiliaan Sobry (maxsobry@gmail.com)
+* **Status:** draft
+
+- [Summary](#summary)
+- [Glossary](#glossary)
+- [Motivation](#motivation)
+- [Requirements and goals](#requirements-and-goals)
+- [High-level design](#high-level-design)
+- [Detailed design](#detailed-design)
+- [Implementation stories](#implementation-stories)
+- [Alternatives](#alternatives)
+
+## Summary
+
+<!--
+This is a proposal for creating a universal Database plugin, than can be used for ingesting data from multiple databases.
+As an result it will make the process easier and faster because there wouldn't be a need for having multiple plugins for each database.
+This plugin will be a all-in-one plugin that can handle ingestion's from multiple databases.
+-->
+
+## Glossary
+<!--
+DAL: Database Abstraction Layer
+
+-->
+
+## Motivation
+<!--
+Implementing a Universal Database Plugin takes away the work of creating a new plugin on every Database the user request.
+Therefore if a new Database should be added for ingestion, this can be implemented faster. Above that having one plugin will result in having a better support, easier management and debugging.
+-->
+
+## Requirements and goals
+<!--
```

Comment:
THis `<!--` and `-->` is a comment in HTML/Markdown

You are supposed to put things in comments that you do not want people to see when rendered. 

Your proposal is something you want people to see , so should be outside of comments